sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -105,13 +112,15 @@ def get_role_assignment(self, scope, name, **kwargs):
         return KeyVaultRoleAssignment._from_generated(assignment)
 
     @distributed_trace
-    def list_role_assignments(self, scope, **kwargs):
-        # type: (Union[str, KeyVaultRoleScope], **Any) -> ItemPaged[KeyVaultRoleAssignment]
+    def list_role_assignments(
+        self, scope: "Union[str, KeyVaultRoleScope]", **kwargs: "Any"
```

Comment:
You can remove hints for kwargs which is the comment I got: https://github.com/Azure/azure-sdk-for-python/pull/28095#discussion_r1060868828

Reply:
That's good to know, thanks for the heads-up! I'll push an update in a sec